main.py

```diff
@@ -38,6 +52,127 @@
     db_format = json.load(file_handle)
 
 
+def file_size(file_path):
```

Comment:
This function does not appear to be used anywhere

Reply:
Gone